package-res/resources/web/prompting/builders/ListBuilder.js

```diff
@@ -15,8 +15,45 @@
  *
  */
 
-define(['cdf/components/SelectComponent', 'cdf/components/SelectMultiComponent',
-    './ValueBasedParameterWidgetBuilder'],
+/**
+ * <h2>The List Builder</h2>
+ *
+ * To use the ListBuilder you should require the appropriate file
+ * from Common-Ui:
+ *
+ * <pre><code>
+ *   require(['common-ui/builders/ListBuilder'],
+ *     function(ListBuilder) {
+ *
+ *     }
+ *   );
+ * </code></pre>
+ *
+ * To get the component you'll have to create a new instance of the builder and
+ * call the <code>build</code> method:
+ *
+ * <pre><code>
+ *   var listBuilder = new ListBuilder();
+ *
+ *   var listComponent = listBuilder.build(args);
+ * </code></pre>
+ *
+ * where 'args' is an object that contains the prompt panel and the parameters
+ * necessary for the component.
+ * The property multiSelect defines what is the component created.
+ * If multiSelect equals <code>true</code>, a {@link http://localhost:8080/pentaho/api/repos/:public:plugin-samples:pentaho-cdf:pentaho-cdf-require:30-documentation:30-component_reference:10-core:16-SelectComponent:select_component.xcdf/generatedContent SelectComponent} is created.
+ * If multiSelect equals <code>false</code>, a {@link http://localhost:8080/pentaho/api/repos/:public:plugin-samples:pentaho-cdf:pentaho-cdf-require:30-documentation:30-component_reference:10-core:19-SelectMultiComponent:select_multi_component.xcdf/generatedContent MultiSelectComponent} is created.
+ *
+ * <p>
+ *   Note: the CDF documentation points to the Dashboard located on the Pentaho BI Server
+ * </p>
+ *
+ * @name ListBuilder
+ * @class
+ * @extends ParameterWidgetBuilderBase
```

Comment:
incorrect parent class name, used ValueBasedParameterWidgetBuilder for extending